aws-lc-fips-sys/builder/printenv.bat

```diff
@@ -1,7 +1,35 @@
 @echo off
+setlocal EnableDelayedExpansion
 for /f "usebackq tokens=* delims=" %%i in (`"%ProgramFiles(x86)%\Microsoft Visual Studio\Installer\vswhere.exe" -latest -property installationPath`) do (
-    set "VS_PATH=%%i"
+    set "VS_PATH=%%i\VC\Auxiliary\Build\"
```

Comment:
we can assume that this directory exists?

Reply:
This is the standard location of the script for [VS 2017 and VS 2019](https://learn.microsoft.com/en-us/cpp/build/building-on-the-command-line?view=msvc-170#vcvarsall) (and VS 2022 apparently) but it doesn't appear to be guaranteed:
> ... Then, use CD again to change to the subdirectory that contains the configuration-specific command files. For Visual Studio 2019 and Visual Studio 2017, use the VC\Auxiliary\Build subdirectory. For Visual Studio 2015, use the VC subdirectory.